app/views/users/show.html.erb

```diff
@@ -0,0 +1 @@
+<h1>welcome to stickit <%= user.username %>!</h1>
```

Comment:
Use translations

Reply:
Will add this in a seperate feature branch 👍